src/benchmark_runner.cc

```diff
@@ -173,18 +171,25 @@ BenchTimeType ParseBenchMinTime(const std::string& value) {
     return ret;
   }
 
-  std::string min_time_str;
-  if (value.back() != 's') {
+  const char* time_str = value.c_str();
+  bool has_suffix = value.back() == 's';
+  if (!has_suffix) {
     BM_VLOG(2) << "Value passed to --benchmark_min_time should have a suffix. "
```

Comment:
if this is something we want users to take action on, we need to log at a higher level than VLOG(2).